arm-software/shared/elf2bin/test/makeelf.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python3
```

Comment:
Isn't this very similar to yaml2obj? 

If we're able to guarantee that the llvm tools build then perhaps the test suite can use yaml2obj and not need this?